src/lambertw.jl

```diff
@@ -0,0 +1,386 @@
+import Base: convert
+#export lambertw, lambertwbp
+using Compat
+
+const euler =
```

Comment:
You can just do
```julia
using Compat.MathConstants: e
```
then use `e` instead of `euler`.

Reply:
Thanks.  I couldn't figure out the compat syntax from the docs, so I just did what I could.